design/src/App.scss

```diff
@@ -77,6 +78,115 @@ $hover-color: #f8cc80;
     }
 }
 
+.glogg-h1 {
+    font-family: 'Merienda', cursive;
+    font-size: 35pt;
+    color: #ffffff;
+    text-align: center;
+}
+
+.glogg-row1{
+    padding: 50px;
+    background-color: #9D8471;
+}
+
+.glogg-ribbon-text{
+    font-family: 'Merienda', cursive;
+    font-size: 18pt;
+    color: #ffffff;
+    text-align: center;
+    position: absolute;
+    width: 8em;
+    height: 10em;
+    z-index: 9999;
+    top: 0;
+    right: 10%;
+    background: $ribbon-color;
+    padding: 5px 0;
+}
+
+.glogg-ribbon{
+    z-index: 99999;
+    position: relative;
+    margin: 0 5px;
+}
+
+.glogg-ribbon-text:before {
+    content: "";
+    position: absolute;
+    bottom: -4em;
+    left: 0;
+    width: 0;
+    height: 0;
+    border-top: 5em solid $ribbon-color;
+    border-right: 5em solid transparent;
+    z-index: -1;
+}
+  
+.glogg-ribbon-text:after {
+    content: "";
+    position: absolute;
+    bottom: -4em;
+    right: 0;
+    width: 0;
+    height: 0;
+    border-top: 5em solid $ribbon-color;
+    border-left: 5em solid transparent;
+    z-index: -1;
+}
+
+.glogg-text{
+    color: #ffffff;
+    font-size: 14pt;
+    font-family: Lato;
+    position: relative;
+    margin-top: 18em;
+    margin: 0;
+    padding: 15px;
+    line-height: 16px;
+    background-color: #25271c; 
```

Comment:
Consider turning color values into variables, as well as fonts, for easy changing.